src/pharmpy/tools/amd/run.py

```diff
@@ -128,6 +128,15 @@ def run_amd(
     if modeltype == 'pkpd':
         dv = 2
         iiv_strategy = 'pd_fullblock'
+        try:
+            input_search_space_features = [] if search_space is None else mfl_parse(search_space)
+        except:  # noqa E722
+            raise ValueError(f'Invalid `search_space`, could not be parsed: "{search_space}"')
+
+        if search_space is None:
+            structsearch_features = "DIRECTEFFECT(*);EFFECTCOMP(*)"
```

Comment:
Should we have indirect effect in the default?

Reply:
I was not sure if we want the default search space to be too large. But I can of course add indirect effects. Will do that in the next commit.